content/en/docs/ambient/getting-started/enforce-auth-policies/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Enforce authorization policies
+description: Enforce Layer 4 and Layer 7 authorization policies in an ambient mesh.
+weight: 4
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+After you have added your application to the ambient mesh, you can secure application access using Layer 4 authorization policies.
+
+This feature lets you control access to and from a service based on the client workload
+identities that are automatically issued to all workloads in the mesh.
+
+## Enforce Layer 4 authorization policy
+
+Let's create an [authorization policy](/docs/reference/config/security/authorization-policy/) that restricts which services can communicate with the `productpage` service. The policy is applied to pods with the `app: productpage` label, and it allows calls only from the the service account `cluster.local/ns/default/sa/bookinfo-gateway-istio`. (This is the service account that is used by the Bookinfo gateway you deployed in the previous step.)
+
+{{< text syntax=bash snip_id=deploy_l4_policy >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: productpage-viewer
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: productpage
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        principals:
+        - cluster.local/ns/default/sa/bookinfo-gateway-istio
+EOF
+{{< /text >}}
+
+If you open the Bookinfo application in your browser (`http://localhost:8080/productpage`), you will see the product page, just as before. However, if you try to access the `productpage` service from a different service account, you should see an error.
+
+Let's try accessing Bookinfo application from a `sleep` pod:
+
+{{< text syntax=bash snip_id=deploy_sleep >}}
+$ kubectl apply -f {{< github_file >}}/samples/sleep/sleep.yaml
+{{< /text >}}
+
+Since the `sleep` pod is using a different service account, it will not have access the `productpage` service:
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- curl -s "http://productpage:9080/productpage"
+command terminated with exit code 56
+{{< /text >}}
+
+## Enforce Layer 7 authorization policy
+
+To enforce Layer 7 policies, you first need a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} for the namespace. This proxy will handle all Layer 7 traffic entering the namespace.
```

Comment:
TODO correct for the fact it doesn't apply for traffic coming from gateways - bug @linsun about hairpinning

Reply:
Do we want to go into more details here or just link to the L7 doc page instead?